itest/assertions.go

```diff
@@ -1038,6 +1038,11 @@ func AssertUniverseRoot(t *testing.T, client unirpc.UniverseClient,
 }
 
 func AssertUniverseRootEqual(a, b *unirpc.UniverseRoot) bool {
+	// Basic RPC form sanity checks.
+	if (a.Id != nil && b.Id == nil) || (a.Id == nil && b.Id != nil) {
```

Comment:
Wouldn't this be taken care of below?

Reply:
We don't check for `a.Id == nil` below this point which can cause a nil pointer dereference panic.